cluster/gce/configure-vm.sh

```diff
@@ -430,31 +442,46 @@ EOF
   fi
 }
 
+function try-download-release() {
+  # TODO(zmerlynn): Now we REALLy have no excuse not to do the reboot
+  # optimization.
+
+  # TODO(zmerlynn): This may not be set yet by everyone (GKE).
```

Comment:
Correction: I think this pull works for that script. What it doesn't do is handle the upgrade/pull case correctly (the `common.sh` path).